sql/core/src/test/resources/sql-tests/inputs/postgreSQL/int8.sql

```diff
@@ -6,15 +6,15 @@
 -- Test int8 64-bit integers.
 -- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/int8.sql
 --
-CREATE TABLE INT8_TBL(q1 bigint, q2 bigint) USING parquet;
+CREATE TABLE INT8_TBL(id int, q1 bigint, q2 bigint) USING parquet;
```

Comment:
Why we need test aginast PostgresSQL? You mentioned your implementation didn't follow PostgresSQL spec?

Reply:
Because cannot preserves stable order when we running SQL without sort.
The id only used to give a definite order.